includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -571,6 +571,11 @@ public function process_payment_for_order( $cart, $payment_information ) {
 		$intent_failed  = false;
 		$payment_needed = $amount > 0;
 
+		// Make sure that we attach the payment method and the customer ID to the order meta data.
+		$payment_method = $payment_information->get_payment_method();
+		$order->update_meta_data( '_payment_method_token', $payment_method );
```

Comment:
I think `_payment_method_token` might create some confusion with the WC tokens. Would you mind changing this to `_payment_method_id` here and in the server PR?